classes/services/queryBuilders/SubmissionListQueryBuilder.inc.php

```diff
@@ -44,35 +44,23 @@ public function appGet($q) {
 		$primaryLocale = \AppLocale::getPrimaryLocale();
 		$locale = \AppLocale::getLocale();
 
-		$this->columns[] = Capsule::raw('COALESCE(stl.setting_value, stpl.setting_value) AS section_title');
-		$this->columns[] = Capsule::raw('COALESCE(sal.setting_value, sapl.setting_value) AS section_abbrev');
+		$this->columns[] = Capsule::raw('aggregation.section_title');
+		$this->columns[] = Capsule::raw('aggregation.section_abbrev');
```

Comment:
@nibou230, where does `aggregation` come from? I would expect to see it specified somewhere else in in the code (unless it's some SQL Server construct I'm not familiar with).

Reply:
Ah, I see it now in the pkp-lib PR. I'll comment over there.